bluepymm/prepare_combos/parse_files.py

```diff
@@ -129,7 +129,7 @@ def read_mm_recipe_yaml(recipe_filename):
     for region in recipe['neurons']:
         for etype in region['traits']['etype'].keys():
             n_combos = len(mecombos)
-            mecombos.loc[n_combos, 'layer'] = region['traits']['layer']
+            mecombos.loc[n_combos, 'layer'] = int(region['traits']['layer'])
```

Comment:
Is this necessary for this test to pass?

I kind of thought bluepymm could handle string layers. Afai remember we're using that for hippocampus.

Reply:
For the test it is not necessary since the test is ignoring the data type of the dataframe. But by default the yaml parser is parsing the value as the `object` type not string. Should we do something about it?